packit_service/worker/build/copr_build.py

```diff
@@ -232,17 +234,40 @@ def run_build(
                 "Copr owner not set. Use Copr config file or `--owner` when calling packit CLI."
             )
 
-        self.api.copr_helper.create_copr_project_if_not_exists(
-            project=self.job_project,
-            chroots=list(self.build_targets),
-            owner=owner,
-            description=None,
-            instructions=None,
-            list_on_homepage=self.list_on_homepage,
-            preserve_project=self.preserve_project,
-            additional_repos=self.additional_repos,
-            update_additional_values=(owner == "packit"),
-        )
+        try:
+            self.api.copr_helper.create_copr_project_if_not_exists(
+                project=self.job_project,
+                chroots=list(self.build_targets),
+                owner=owner,
+                description=None,
+                instructions=None,
+                list_on_homepage=self.list_on_homepage,
+                preserve_project=self.preserve_project,
+                additional_repos=self.additional_repos,
+            )
+        except PackitCoprSettingsException as ex:
+            if self.metadata.pr_id:
+
+                table = (
+                    "| field | old value | new value |\n"
+                    "| ----- | --------- | --------- |\n"
+                )
+                for field, (old, new) in ex.fields_to_change.items():
+                    table += f"| {field} | {old} | {new} |\n"
+
+                msg = (
+                    "We need to change settings of your copr project "
+                    "but we are not allowed to do so.\n\n"
+                    "You have two options to fix your build:\n"
+                    "- Give us `admin` rights to your Copr Project. "
+                    "(We have just asked asked for it.)\n"
+                    "- Change the following settings manually\n\n"
+                    f"{table}\n"
+                    "Please, retrigger the build once you fix the issues."
+                )
+                self.project.pr_comment(pr_id=self.metadata.pr_id, body=msg)
```

Comment:
now that I look at this line, we totally need the token management magic to have in ogr